src/encrypt.js

```diff
@@ -136,14 +136,19 @@ function createInfo(type, context) {
   const l = type.length;
   const info = new Buffer(18 + l + 1 + 5 + 135);
 
-  info.write('Content-Encoding: ');   // 18 bytes (18 total)
-  info.write(type, 18);               // l bytes (18 + l total)
-  info.write('\0', 18 + l);           // 1 byte (19 + l total)
-  info.write('P-256', 19 + l);        // 5 bytes (24 + l total)
-  context.copy(info, 24 + l);         // 135 bytes (159 + l total)
+  // 18 bytes (18 total)
```

Comment:
Personal style time again. The notes on the sizes are intended to be like marginal notes. They don't describe what the code does, just annotate some extra information. In that context, splitting the lines apart with separating comments makes this code (for me) less readable.

I do seem to remember that this is one of the ESLint rules though, in which case I would rewrite the comment altogether:

```
// The start index for each element within the buffer is:
// value               | length | start  |
// ---------------------------------------
// 'Content-Encoding: '|   18   | 0      |
// type                |   l    | 18     |
// nul byte            |   1    | 18 + l |
// 'P-256'             |   5    | 19 + l |
// info                |   135  | 24 + l |

info.write('Content-Encoding: ');
info.write(type, 18);
info.write('\0', 18 + l);
info.write('P-256', 19 + l);
context.copy(info, 24 + l);
```